tests/test_models.py

```diff
@@ -244,6 +244,15 @@ def test_get_search_queryset(self):
     def test_in_search_queryset(self, mock_qs):
         """Test the in_search_queryset method."""
         obj = ExampleModel(id=1)
+        ExampleModel.objects.in_search_queryset(obj.pk)
+        mock_qs.assert_called_once_with(index="_all")
+        mock_qs.return_value.filter.assert_called_once_with(pk=1)
+        mock_qs.return_value.filter.return_value.exists.assert_called_once_with()
+
+    @mock.patch.object(ExampleModelManager, "get_search_queryset")
+    def test_in_search_queryset_with_a_model_using_custom_primary_key(self, mock_qs):
+        """Test the in_search_queryset method."""
+        obj = ExampleModelWithCustomPrimaryKey(simple_field_1=1)
         ExampleModel.objects.in_search_queryset(obj.id)
```

Comment:
Ironically this failed CI because of the old `.id` attr ;-)

```suggestion
        obj = ExampleModelWithCustomPrimaryKey(simple_field_1=1)
        ExampleModel.objects.in_search_queryset(obj.pk)
```

Reply:
🤦‍♂️ Thanks for calling this out. I've pushed a fixup commit (b859a3ffc29d100c0498f5bf26d8da41cf66868f)